test/known_issues/test-esm-loader-cache-clearing.js

```diff
@@ -0,0 +1,10 @@
+// Flags: --experimental-modules
+"use strict";
+
+const { cache } = require;
+
+require("../common");
```

Comment:
Nit: Can you move this line to be the first line after `"use strict";`. It checks for global variable leaks and so it should be the first thing loaded before anything else happens. (There might or might not be an existing lint rule that checks this. If so, it might or might not be broken.)